resources/images/errors/401-dark.svg

```diff
@@ -0,0 +1 @@
2-29.018v83.354h15.145v19.28h-15.145v32.329h-22.912V180.75H190v-16.749c2.589-5.712 6.019-12.269 10.291-19.67 4.401-7.4 9.255-14.995 14.562-22.785a401.226 401.226 0 0 1 16.893-22.981c5.954-7.53 11.908-14.347 17.863-20.449h22.329ZM405.886 145.5c0 22.721-4.143 40.184-12.427 52.388-8.284 12.075-19.74 18.112-34.367 18.112-14.628 0-26.083-6.037-34.368-18.112-8.284-12.204-12.426-29.667-12.426-52.388 0-11.296 1.035-21.293 3.106-29.992 2.201-8.828 5.307-16.229 9.32-22.201 4.142-5.973 9.061-10.517 14.757-13.633C345.306 76.558 351.843 75 359.092 75c14.627 0 26.083 6.102 34.367 18.307 8.284 12.074 12.427 29.472 12.427 52.193Zm-24.271 0c0-7.011-.388-13.568-1.165-19.67-.777-6.102-2.071-11.36-3.883-15.775-1.683-4.544-3.949-8.114-6.796-10.711-2.848-2.597-6.408-3.895-10.679-3.895-4.272 0-7.896 1.298-10.874 3.895-2.847 2.597-5.177 6.167-6.99 10.711-1.682 4.415-2.912 9.673-3.689 15.775-.647 6.102-.971 12.659-.971 19.67 0 7.011.324 13.568.971 19.67.777 6.102 2.007 11.425 3.689 15.97 1.813 4.414 4.143 7.919 6.99 10.516 2.978 2.597 6.602 3.895 10.874 3.895 4.271 0 7.831-1.298 10.679-3.895 2.847-2.597 5.113-6.102 6.796-10.516 1.812-4.545 3.106-9.868 3.883-15.97.777-6.102 1.165-12.659 1.165-19.67Zm50.164-39.34c7.508-2.986 15.016-6.816 22.524-11.49 7.637-4.674 14.368-10.192 20.193-16.554H491v134.963h-23.494V109.666c-3.495 2.726-7.896 5.388-13.203 7.985-5.308 2.596-10.421 4.739-15.339 6.426l-7.185-17.917Z" stroke="#8692FF" stroke-width="2"/><rect x="58" y="29" width="20" height="20" rx="10" fill="#23272D" stroke="#8692FF" stroke-width="2"/><rect x="417" y="29" width="20" height="20" rx="10" fill="#23272D" stroke="#8692FF" stroke-width="2"/><rect x="477" y="330" width="10" height="10" rx="5" fill="#23272D" stroke="#8692FF" stroke-width="2"/><rect x="303" y="306" width="10" height="10" rx="5" fill="#23272D" stroke="#8692FF" stroke-width="2"/><g clip-path="url(#clip1_23951_1255)"><path d="m644.142 182.226 28.029 7.51a4 4 0 0 0 4.312-1.57l17.638-25.198a3.999 3.999 0 0 0-.015-4.608l-16.971-23.921a4.005 4.005 0 0 0-2.227-1.55l-27.806-7.45a3.999 3.999 0 0 0-4.296 1.547l-17.756 24.993a4 4 0 0 0-.015 4.612l16.867 24.067a4.007 4.007 0 0 0 2.24 1.568Z" fill="#23272D"/><path d="m664.986 152.646 27.924 7.482a2 2 0 0 1 1.116 3.086l-18.42 26.088a2 2 0 0 1-2.151.778l-27.925-7.482a2 2 0 0 1-1.116-3.086l18.42-26.088a2.001 2.001 0 0 1 2.152-.778Z" fill="#8692FF"/><path d="m644.142 182.226 28.029 7.51a4 4 0 0 0 4.312-1.57l17.638-25.198a3.999 3.999 0 0 0-.015-4.608l-16.971-23.921a4.005 4.005 0 0 0-2.227-1.55l-27.806-7.45a3.999 3.999 0 0 0-4.296 1.547l-17.756 24.993a4 4 0 0 0-.015 4.612l16.867 24.067a4.007 4.007 0 0 0 2.24 1.568Zm-1.405-.377 20.924-29.785m31.821 8.526-31.821-8.526m0 0-19.067-27.093" stroke="#8692FF" stroke-width="2"/></g><defs><clipPath id="clip0_23951_1255"><rect width="134" height="181" fill="white" transform="matrix(-1 0 0 1 136 61)"/></clipPath><clipPath id="clip1_23951_1255"><rect width="76" height="67" fill="white" transform="translate(604 107)"/></clipPath></defs></svg>
```

Comment:
instead of duplicating images for dark mode, can you have a look at variables instead like we do for other projects. For example in `laravel-foundation` we make use of `stroke="var(--icon-error-secondary-300, #DBDEE5)"` for the dark mode error images, where we define the variable based on dark/light mode so it reflects it immediately and the fallback is the default light mode color